lib/DDG/Spice/RxInfo.pm

```diff
@@ -6,24 +6,27 @@ use DDG::Spice;
 
 name "RxInfo";
 source "C3PI RxImageAccess RESTful API";
-description "Search for prespection and over the counter solid dosage pharmaceutical by NDC and characteristics (shape, color, imprint).";
+description "Search for prescription and over the counter solid dosage pharmaceutical by NDC and characteristics (shape, color, imprint).";
 primary_example_queries "pill red round", "rxinfo 68180-0481-01", "pill LL blue";
 category "special";
 topics "everyday", "special_interest";
 code_url "https://github.com/duckduckgo/zeroclickinfo-spice/blob/master/lib/DDG/Spice/RxInfo.pm";
-attribution web => ['http://www.medicosconsultants.com', 'Medicos Consultants, LLC'];
+attribution web => ['http://www.medicosconsultants.com', 'Medicos Consultants, LLC'],
+            twitter => ['jlanza2', 'Joseph Lanza'];
 
 triggers startend => 'pill', 'rxinfo', 'capsule', 'tablet', 'softgel', 'caplets';
 
 spice to => 'http://rximage.nlm.nih.gov/api/rximage/1/rxbase?resolution=300&includeIngredients=true&parse=$1';
-spice from => '(.*?)/(\d)';
-spice wrap_jsonp_callback => 1;
 
-my $triggerWords = join "|", share('triggerWords.txt')->slurp(chomp => 1);
+spice wrap_jsonp_callback => 1;
 
 handle remainder => sub {
-    return unless $_;
-    return $_, (/$triggerWords/) ? "0" : "1"; # return 0 if match in triggerWords file found
+    # Remove logical operators from query since they're disregarded by the API.
+    $_ =~ s/and//ig;
+    $_ =~ s/or//ig;
+    $_ =~ s/not//ig;
```

Comment:
These can be done on a single line:

`s/and|or|not//ig` there's also no need to include `$_ =~`